src/interface/src/app/plan/plan-map/plan-map.component.ts

```diff
@@ -22,7 +23,7 @@ export class PlanMapComponent implements AfterViewInit, OnDestroy {
   ngAfterViewInit(): void {
     if (this.map != undefined) this.map.remove();
 
-    this.map = L.map('map', {
+    this.map = L.map(this.mapId ? this.mapId : 'map', {
```

Comment:
Out of curiousity, how does mapId work when reusing maps?

Reply:
I was running into a bug I couldn't diagnose (leaflet simply wouldn't initialize the map, silent w/ no errors) when I did the following:
1. PlanMapComponent.ngAfterViewInit() => initialize map with id "map"
2. PlanMapComponent.ngOnDestroy() => map.remove()
3. <user navigates from plan summary to scenario planning w/ different map container but reused component)
4. PlanMapComponent.ngAfterViewInit() => initialize map with id "map"

But it was fixed when I used different map IDs for steps 1 and 4. I'm not sure why!